### PR TITLE
chore: foundation for gnosocial mobile app

### PR DESCRIPTION
**Pull request by iuricmp**
This PR creates a basic foundation for the new mobile app for GnoSocial.
The interaction with gnosocial realms and the expected functionalities will be done in the following PR.

### CODE REVIEW

File: .DS_Store
Comment (jefft0) [inline .DS_Store:1]: Should remove this file and update .gitignore with:
```
.DS_Store
```

**Comment by D4ryl00** [comment]
I tested your latest commit (374f17eae9131fda06bccf18aaf8f8197663b817) and that works for Android. I can create an account, navigate to different screens.

**Review by D4ryl00 — APPROVED** [review]
Tested on iOS and Android. LGTM for a first iteration.

**Review by jefft0 — APPROVED** [review]
Tested on Android and iOS servers.

**Pull request merged**